app/helpers/schools/dashboard_helper.rb

```diff
@@ -25,5 +29,14 @@ def mentor_count(school_cohorts)
     def participants_count(school_cohorts)
       school_cohorts.sum { |sc| sc.current_induction_records.count }
     end
+
+    def missing_mentor_html(participant_profile)
+      tag.div(class: "govuk-summary-list__missing") do
+        concat tag.div("No mentor assigned", class: "govuk-summary-list__missing-heading")
+        concat govuk_link_to("Assign a mentor",
+                             school_participant_edit_mentor_path(participant_id: participant_profile.id),
+                             no_visited_state: true)
+      end
```

Comment:
My bad - we should use the `app-` prefix here for our custom styles.

```suggestion
    def missing_mentor_html(participant_profile)
      tag.div(class: "app-summary-list__missing") do
        concat tag.div("No mentor assigned", class: "app-summary-list__missing-heading")
        concat govuk_link_to("Assign a mentor",
                             school_participant_edit_mentor_path(participant_id: participant_profile.id),
                             no_visited_state: true)
      end
```